File: tests/testdata/codegen/string-enum-union-list-pp/go/string-enum-union-list.go
package main

import (
	"github.com/pulumi/pulumi-azure-native/sdk/go/azure/servicebus"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		_, err := servicebus.NewNamespaceAuthorizationRule(ctx, "namespaceAuthorizationRule", &servicebus.NamespaceAuthorizationRuleArgs{
			AuthorizationRuleName: "sdk-AuthRules-1788",
			NamespaceName:         pulumi.String("sdk-Namespace-6914"),
			ResourceGroupName:     pulumi.String("ArunMonocle"),
			Rights: pulumi.StringArray{
				pulumi.String(servicebus.AccessRightsListen),
				pulumi.String(servicebus.AccessRightsSend),
			},
		})
		if err != nil {
			return err
		}
		return nil
	})
}

File: pkg/codegen/testing/utils/testdata/string-enum-union-list-pp/python/string-enum-union-list.py
import pulumi
import pulumi_azure_native as azure_native

namespace_authorization_rule = azure_native.servicebus.NamespaceAuthorizationRule("namespaceAuthorizationRule",
    authorization_rule_name="sdk-AuthRules-1788",
    namespace_name="sdk-Namespace-6914",
    resource_group_name="ArunMonocle",
    rights=[
        azure_native.servicebus.AccessRights.LISTEN,
        azure_native.servicebus.AccessRights.SEND,
    ])
